src/components/PatientProfile/PatientProfilePage.tsx

```diff
@@ -435,7 +418,7 @@ const PatientProfilePage = ({
                 notes: (e.target as HTMLInputElement).value,
               });
             }}
-            value={formFields.notes}
+            value={formFields.notes ? formFields.notes : ''}
```

Comment:
nit: this can also be simplified to the following - either option is fine and is a stylistic choice, so up to you if you want to make this change
```suggestion
            value={formFields.notes || ''}
```